src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs

```diff
@@ -34,7 +34,6 @@ public class DurableTaskOptions
         /// </summary>
         public TraceOptions Tracing { get; set; } = new TraceOptions();
 
-

```

Comment:
We may want to revert this change.

Reply:
Hmm, but I think the code wouldn't compile with two blank lines in a row, right?

Reply:
Ah, that makes sense.